test/verify/check-system-journal

```diff
@@ -222,6 +222,11 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
         wait_log_present("JUST ERROR")
         wait_log_present("THE SERVICE")
 
+        # HACK: https://github.com/cockpit-project/cockpit/issues/20123
+        # reload the page to reload "Indetifiers" list, to avoid issue that list of services is not valid
```

Comment:
If you fixup the commit message there is a typo here `Identifiers`

Reply:
thanks a lot , fixed your mentioned comments.